modules/kafka/kafka.go

```diff
@@ -155,12 +215,43 @@ func copyStarterScript(ctx context.Context, c testcontainers.Container) error {
 	return nil
 }
 
-func WithClusterID(clusterID string) testcontainers.CustomizeRequestOption {
-	return func(req *testcontainers.GenericContainerRequest) error {
-		req.Env["CLUSTER_ID"] = clusterID
+func editEnvsForListeners(listeners []KafkaListener) map[string]string {
+	if len(listeners) == 0 {
+		// no change
+		return map[string]string{}
```

Comment:
suggestion: return nil as its just used as an iterator
```suggestion
		return nil
```

Reply:
I think we should apply this values in the functional option. Wdyt?

Reply:
Yes, we should always try to have standalone functional options which act directly on the request if possible, makes things much cleaner and also allows further options to override if needed.